measure_extinction/extdata.py

```diff
@@ -880,16 +912,18 @@ def read(self, ext_filename):
         for curkey in column_keys:
             if pheader.get(curkey):
                 if pheader.get("%s_UNC" % curkey):
-                    tunc = float(pheader.get("%s_UNC" % curkey))
+                    tuple = float(pheader.get(curkey)), float(
```

Comment:
Why call this "tuple"?  It is one, but that does make this not a great name.  Why not keep "tunc"?

Reply:
Fixed!